docs/howto/operate/ci-cd.md

```diff
@@ -0,0 +1,138 @@
+# Our CI/CD system
+
+The best place to look to learn about the latest state of our *automatic* hub deployment
+is to look at [the `deploy-hubs.yaml` GitHub workflow](https://github.com/2i2c-org/pilot-hubs/blob/master/.github/workflows/deploy-hubs.yaml).
+That workflow defines the automatic hub deployment for many of our major clusters.
+
+:::{note}
+This is essentially a placeholder, it needs to be extended to talk a LOT more about our
+CI/CD system.
+:::
+
+## AWS cluster/hubs specific requirements
+
+The pilot-hubs deployer code supports [a new `auth_aws` function](https://github.com/2i2c-org/pilot-hubs/blob/e96e7bcded187870dc2e07d6626de8a12586ed32/deployer/hub.py#L126)
+to get a temporary kubeconfig file that allows a successful connection with the
+kubernetes cluster.
+It is actually expecting a 2i2c generated IAM entity (the `deployer` user) and its
+AWS credentials (that we provide in a sops encrypted json file) to properly set them up
+as the AWS `Access Key ID` and `Secret Access Key` as environment variables and
+successfully perform operations in the cluster as the `deployer` user.
+
+There are some requisites for being able to successfully and automatically deploy hubs
+on AWS clusters using the provided functionality:
+
+1. Know if you are working with a [AWS EKS-based](https://aws.amazon.com/eks/) or a
+[kops-based](https://kops.sigs.k8s.io/getting_started/aws/) cluster
+  
+* We currently deploying `EKS`-based (ie, Carbonplan) and `kops`-based cluster (ie, Farallon).
+Whether to create one or another depends on the cluster operational needs and some other
+factors outlined (and being discussed on [this issue](https://github.com/2i2c-org/pilot-hubs/issues/431)). 
+You need to know and understand which sort of cluster are you working with to properly
+set up the config file in step 4.
+
+Look at [this section](new-cluster:aws) if you are interested in setting up a `kops`-based
```

Comment:
Can we use more descriptive words than "this section"? E.g. `Look at [the AWS cluster setup instructions](link) if you are interested...`